content/_data/changelogs/lts.yml

```diff
@@ -3518,6 +3518,95 @@
     message: |-
       Developer: Prevent spurious deprecation messages by removing the deprecated findbugs-annotation.
 
+- version: "2.222.4"
+  date: 2020-05-24
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4479
+    issue: 61202
+    authors:
+    - timja
+    message: |-
+      Users with extended read permission now get a more read-only looking UI.
+  - type: major bug
```

Comment:
Hmm does this need mentioning? It was caused by the one above, and it’s certainly not a major bug for the LTS as the bug never occurred

Reply:
It shouldn't be mentioned to prevent confusion.

The one below that is arguable, IIRC it could technically occur.

Reply:
Thanks for the feedback.  I've removed it as a separate entry and included it as a reference in another entry